evm-contracts/src/v0.7/dev/OperatorProxy.sol

```diff
@@ -0,0 +1,18 @@
+pragma solidity 0.7.0;
+
+import "./Owned.sol";
+import "../interfaces/OracleInterface.sol";
+import "./LinkTokenReceiver.sol";
+
+contract OperatorProxy is Owned {
+
+  constructor() Owned(msg.sender) {}
+
+  function forward(address _to, bytes calldata _data) public
+  {
+    require(OracleInterface(owner).isAuthorizedSender(msg.sender), "Not an authorized node");
+    require(_to != LinkTokenReceiver(owner).getChainlinkToken(), "Cannot send to Link token");
```

Comment:
Either in this PR or the follow up, we should set the LINK token address as an immutable field on here. That will save ~1500 gas on every call.